tests/test_models.py

```diff
@@ -128,15 +140,8 @@ def test_rounding(self):
             (Q(amount1=Money(1, 'USD')) | Q(amount1__gte=Money(4, 'USD'), amount2=Money(0, 'GHS')), 2),
         )
     )
+    @pytest.mark.usefixtures('objects_setup')
```

Comment:
I think I'll refactor whole querying tests with single objects setup. But it can wait, and I hope, that such approach will simplify our tests